src/popup/router/pages/NamingSystem.vue

```diff
@@ -219,16 +250,25 @@ export default {
       this.bySoonest = true;
       this.byCharLength = false;
       this.byBid = false;
+      this.byMine = false;
+    },
+    filterByMine() {
+      this.bySoonest = false;
+      this.byCharLength = false;
+      this.byBid = false;
+      this.byMine = true;
     },
     filterByCharLength() {
       this.byCharLength = true;
       this.byBid = false;
       this.bySoonest = false;
+      this.byMine = false;
     },
     filterByBid() {
       this.byBid = true;
       this.byCharLength = false;
       this.bySoonest = false;
+      this.byMine = false;
     },
```

Comment:
We can optimize these filters by having a single method with a switch or some enum?